erpnext/accounts/doctype/sales_invoice/sales_invoice.py

```diff
@@ -13,7 +13,7 @@
 from erpnext.stock.doctype.delivery_note.delivery_note import update_billed_amount_based_on_so
 from erpnext.projects.doctype.timesheet.timesheet import get_projectwise_timesheet_data
 from erpnext.assets.doctype.asset.depreciation \
-	import get_disposal_account_and_cost_center, get_gl_entries_on_asset_disposal
+	import get_disposal_account_and_cost_center, get_gl_entries_on_asset_movement
```

Comment:
`get_gl_entries_on_asset_movement` can be named `get_gl_entries_on_asset_sale`?

The problem with `get_gl_entries_on_asset_movement` is that, `asset_movement` is another part of Assets that don't involve `gl_entries`, so it can cause confusion.

You could rename the method, or use the same but introduce an argument like `is_return=True`

Reply:
But `get_gl_entries_on_asset_sale` doesn't make sense here and neither does the old name, because we're now considering two different cases here- asset disposal and asset regain. `get_gl_entries_on_asset_sale` would imply that only the former is being handled there. But I agree, `asset_movement` can be confusing, we should think of a better name.

Reply:
`asset disposal and asset regain` sounds good

Reply:
Done.